### DISCUSSION TITLE
bdb文件夹磁盘占用逐渐增大，不能主动删除

### DISCUSSION THREAD

**Discussion by hongzhenfu** [apache/doris#20494] [A - General / Q&A]
Doris Version: 1.1.0-rc05-Unknown
架构： 1个master，3个数据节点，master和数据节点上混部署

FE节点下的image文件夹中的image能够成功生成，bdb文件夹里面的jdb文件不能自动删除，逐渐膨胀；
![image](https://github.com/apache/doris/assets/8897241/cf8fd27c-1ba2-4b88-9f7c-385583a52c61)

有没有遇到同样问题的

**Comment by vinlee19** [comment]
It may be because the Master FE is not pushing the image successfully. You can search for push image.xxxx to other nodes in the master FE's fe.log. totally xx nodes, push successed yy nodes. if yy is smaller than xx, then some FEs were not pushed successfully. You can see the specific error Exception when pushing image file. url = xxx in fe.log.

Also, you can add a configuration to the FE configuration file: edit_log_roll_num=xxxx. This parameter sets the number of metadata journals to do an image for every journal. the default is 50000. you can change this number to a smaller number to make the images more frequent and thus speed up the deletion of old journals.

**Comment by hongzhenfu** [comment]
是版本的问题，1.1.0的bug，在1.1.2版本解决了

![image](https://github.com/apache/doris/assets/8897241/f286c349-9f92-446c-9d63-54288cf75bed)
